tests/__fixtures__/testGetFixedTFunction/namespace.js

```diff
@@ -0,0 +1,38 @@
+import i18next from "i18next";
+
+export function anyJSFunction1() {
+    const t = i18next.getFixedT(null, 'ns0');
```

Comment:
Could you add:
- a test that uses a comment hint.
- a test with a global t function or at least a t function from an outer scope. I'm not 100% sure this one would pass btw.